lib/write.go

```diff
@@ -72,6 +72,14 @@ func (a *ACBuild) Write(output string, overwrite, sign bool, gpgflags []string)
 	}
 	defer ofile.Close()
 
+	defer func() {
+		// When write is done, if an error is encountered remove the partial
+		// ACI that had been written.
+		if err != nil {
+			os.RemoveAll(output)
```

Comment:
So it is! I removed the removes at 90/98, added a removal for `output + "asc."` here, and switched to Remove here. In this context, Remove and RemoveAll are equivalent afaik.